mmocr/models/textdet/postprocess/wrapper.py

```diff
@@ -391,3 +393,169 @@ def textsnake_decode(preds,
             boundaries.append(boundary + [score])
 
     return boundaries
+
+
+def fcenet_decode(
+    preds,
+    fourier_degree,
+    reconstr_points,
+    scale,
+    tcl_tr_ratio,
+    text_repr_type='poly',
+    tr_thresh=0.8,
+    nms_thresh=0.1,
+):
+    """Decoding predictions of FCEnet to instances.
+
+    Args:
+        preds (list(Tensor)): The head output tensors.
+        fourier_degree (int): The maximum fourier transform degree k
+        reconstr_points (int): The points number of the polygon reconstructed from predicted
+            fourier coefficients.
+        scale (int): The downsample scale of preds
+        tcl_tr_ratio (float): The ratio to calculate final score.
+        text_repr_type (str): boundary encoding type 'poly' or 'quad'.
+        tr_thresh (float) : The threshold used to filter out the final
+            candidates.
+        nms_thresh (float) : The threshold of nms.
```

Comment:
```suggestion
        preds (list(Tensor)): The head output tensors.
        fourier_degree (int): The maximum fourier transform degree k
        reconstr_points (int): The points number of the polygon reconstructed from predicted
            fourier coefficients.
        scale (int): The downsample scale of preds
        tcl_tr_ratio (float): The ratio to calculate final score.
        text_repr_type (str): Boundary encoding type 'poly' or 'quad'.
        tr_thresh (float) : The threshold to filter out the final
            candidates.
        nms_thresh (float) : The threshold of nms.
```
pls rename tcl_tr_ratio